### ISSUE TITLE
ExperimentalAcrylicBorder gives an error

### ISSUE THREAD

**Issue by datstat-consulting** [AvaloniaUI/MusicStoreTutorial#1]
Inserting this block into `MainWindow.axaml`:
```
<Panel>
           <ExperimentalAcrylicBorder IsHitTestVisible="False">
               <ExperimentalAcrylicBorder.Material>
                   <ExperimentalAcrylicMaterial
                       BackgroundSource="Digger"
                       TintColor="Black"
                       TintOpacity="1"
                       MaterialOpacity="0.65" />
               </ExperimentalAcrylicBorder.Material>
             </ExperimentalAcrylicBorder>
</Panel>
```
gives the following error on Visual Studio:
> Unable to find a setter that allows multiple assignments to the property Content of type Avalonia.Controls:Avalonia.Controls.ContentControl Line 1, position 2.

**Comment by maxkatz6** [comment]
We need full MainWindow code. 
Most likely, your Window has multiple children nodes.

**Comment by datstat-consulting** [comment]
Here is the code
```
<Window xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:vm="using:Avalonia.MusicStore.ViewModels"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:views="clr-namespace:Avalonia.MusicStore.Views"
        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
        x:Class="Avalonia.MusicStore.Views.MainWindow"
        Icon="/Assets/avalonia-logo.ico"
        Title="Avalonia.MusicStore"
		
		TransparencyLevelHint="AcrylicBlur"
        Background="Transparent"
		ExtendClientAreaToDecorationsHint="True">
	<views:MainView />
	<Panel>
		<ExperimentalAcrylicBorder IsHitTestVisible="False">
			<ExperimentalAcrylicBorder.Material>
				<ExperimentalAcrylicMaterial
					BackgroundSource="Digger"
					TintColor="Black"
					TintOpacity="1"
					MaterialOpacity="0.65" />
			</ExperimentalAcrylicBorder.Material>
		</ExperimentalAcrylicBorder>
	</Panel>

</Window>
```

**Comment by maxkatz6** [comment]
```
<Window xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:vm="using:Avalonia.MusicStore.ViewModels"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:views="clr-namespace:Avalonia.MusicStore.Views"
        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
        x:Class="Avalonia.MusicStore.Views.MainWindow"
        Icon="/Assets/avalonia-logo.ico"
        Title="Avalonia.MusicStore"
		
		TransparencyLevelHint="AcrylicBlur"
        Background="Transparent"
		ExtendClientAreaToDecorationsHint="True">
	<Panel>
		<ExperimentalAcrylicBorder IsHitTestVisible="False">
			<ExperimentalAcrylicBorder.Material>
				<ExperimentalAcrylicMaterial
					BackgroundSource="Digger"
					TintColor="Black"
					TintOpacity="1"
					MaterialOpacity="0.65" />
			</ExperimentalAcrylicBorder.Material>
		</ExperimentalAcrylicBorder>
         	<views:MainView />
	</Panel>
</Window>
```

**Comment by datstat-consulting** [comment]
For those chancing upon this issue, here's how I solved all problems with the Music Store tutorial.
Run `dotnet new install Avalonia.Templates` even if you're on Visual Studio, the MVVM template there is what's most friendly. I was able to configure the other tutorials with the Visual Studio Avalonia extension's MVVM template, but it's much more convenient either way to use the `Avalonia.Templates` one.
